src/main/pages/setup-docker/docker-single-user.adoc

```diff
@@ -287,7 +298,7 @@ docker run -p 8080:8080 \
 
 *Ports*
 
-Tomcat inside the container will bind itself to port 8080 by default. You must map this port to be exposed in your container using `-p 8080:8080`. If you want to change the port at which your browsers connect, then change the first value, such as `p 9000:8080`. This will route requests from port 9000 to the internal Tomcat bound to port 8080. If you want to change the internal port that Tomcat is bound, you must update the port binding and set `CHE_PORT` to the new value.
+Tomcat inside the container will bind itself to port `8080` by default. You must map this port to be exposed in your container using the `-p 8080:8080` command. If you want to change the port on which your browsers connect, change the first value, such as `p 9000:8080`. This will route requests from port `9000` to the internal Tomcat bound to port `8080`. If you want to change the internal port that Tomcat is bound to, you must update the port binding and set the `CHE_PORT` variable to the new value.
```

Comment:
* the `-p 8080:8080` command -> the `-p 8080:8080` parameter
* on which -> to which
* -> such as `-p 9000:8080`
* This will route -> This routes
* -> To change the internal port...